### ISSUE TITLE
Document to analyse the difference between mask rcnn and detectron2

### ISSUE THREAD

**Issue by tamyxgopenx** [facebookresearch/detectron2#894]
## 🚀 Feature
Document to analyse the difference between mask rcnn and detectron2
## Motivation
Lately i take my time to research for object detection and finetuning both mask rcnn and detectron2, I was very interesting why detectron2 gain better accuracy, is there any different in structure or hyperameter. For example is there any differences in loss/regularization/many other hyperparameters. it's either the architecture difference or implementation

**Comment by ppwwyyxx** [comment]
The differences are in https://detectron2.readthedocs.io/notes/compatibility.html

**Comment by tamyxgopenx** [comment]
@ppwwyyxx Thanks for all your hard work, just one more question: beside the different implementation of pytorch and tensorflow is there anything new between detectron and default matterport mask rcnn?

**Comment by ppwwyyxx** [comment]
We don't know much about it.

**Comment by tamyxgopenx** [comment]
@ppwwyyxx how about detectron2 and mask rcnn matterport

**Comment by ppwwyyxx** [comment]
It's answered above

**Comment by tamyxgopenx** [comment]
@ppwwyyxx  since i watched all the document and either tensorpark, it's all the same structure right (maskrcnn matterport, tensorpark, detectron)?
Detectron2 will be a little bit difference just as the document you provided (the document only compared with detectron which use mask rcnn benchmark - not mask rcnn of matterport)?

**Comment by ivojuroro** [comment]
> Detectron2 will be a little bit difference just as the document you provided (the document only compared with detectron which use mask rcnn benchmark - not mask rcnn of matterport)?

You could take a look at the differences of between matterport implementation and the original paper, those are in the project https://github.com/matterport/Mask_RCNN#differences-from-the-official-paper
